### ISSUE TITLE
[Bug Report] Reverse proxy error

### ISSUE THREAD

**Issue by mbhkoay** [stashapp/stash#2224]
Describe the bug
Videos do not load while browsing from reverse proxy url

To Reproduce
1. works as expected while browsing from xxx.xxx.xxx.xxx:port
2. Able to access into stash, but when browsing from scenes, all videos do not load, showing error 224003 
3. Logs show: error writing to UDP socket: write udp 0.0.0.0:1900->239.255.255.250:1900: sendto: required key not available

Stash Version: (from Settings -> About):
v0.12.0

OS: Unraid
Browser: Both Chrome & Edge
Version 0.12

Additional info:
1. Reverse proxy is using Nginx Proxy Manager
2. dangerous_allow_public_without_auth: true
3. security_tripwire_accessed_from_public_internet: ""

![image](https://user-images.githubusercontent.com/29805565/148667983-a6e513e9-0068-4828-83dd-cbfd02344804.png)
![image](https://user-images.githubusercontent.com/29805565/148668011-cdddebb4-ed0f-409b-986c-c966d3584d3b.png)
![image](https://user-images.githubusercontent.com/29805565/148668019-1528a90c-ca1d-4933-aafb-697f3f816d26.png)

**Comment by kermieisinthehouse** [comment]
This is a known issue with nginx proxy manager on a non-standard port. Regular nginx with plain configuration is unaffected. A fix will be proposed soon.

Known temporary workaround, courtesy of GsMumbo:
"Update - reading over the chat and looking at that code, I decided to add a custom location for / and set the X-Forwarded-Port to 443. Mainly because I saw it was one of the values that was set to be ignored in the code."

![IMG_3450](https://user-images.githubusercontent.com/84814418/148668303-6bc9d5d6-0632-410d-ab87-b7676ea17055.png)
